components/gcp-click-to-deploy/src/DeployForm.tsx

```diff
@@ -262,12 +271,53 @@ export default class DeployForm extends React.Component<any, DeployFormState> {
       if (p.name === 'acmeEmail') {
         p.value = email;
       }
-    });
+
+      if (p.name === 'jupyterHubAuthenticator') {
```

Comment:
So the client side JS is responsible for setting the authenticator for JupyterHub?

Reply:
Ideally not, will take care in https://github.com/kubeflow/kubeflow/issues/1971